includes/class.llms.cache.helper.php

```diff
@@ -31,6 +31,35 @@ public function __construct() {
 
 	}
 
+	/**
+	 * Sets a browser cookie that tells WP Engine to exclude a page from server caching.
+	 *
+	 * @see https://wpengine.com/support/cache/#Default_Cache_Exclusions
+	 * @see https://wpengine.com/support/determining-wp-engine-environment/
+	 *
+	 * @since [version]
+	 *
+	 * @param int|WP_Post $post Optional. Post ID or post object. Default is the global `$post`.
+	 *
+	 * @return void
+	 */
+	private function exclude_page_from_wpe_server_cache( $post = null ) {
+
+		if ( function_exists( 'is_wpe' ) && is_wpe() ) {
+			/*
+			 * If "Settings -> Permalinks" is "Plain", i.e. the `permalink_structure` option is '',
+			 * allow the entire site to be cached by WP Engine.
+			 * Note: This will prevent users from being able to successfully use the "Lost your password?" feature.
+			 */
+			if ( isset( $GLOBALS['wp_rewrite'] ) && '' === $GLOBALS['wp_rewrite']->permalink_structure ) {
```

Comment:
```suggestion
			if ( isset( $GLOBALS['wp_rewrite'] ) && ! $GLOBALS['wp_rewrite']->using_permalinks() ) {
```

I think using `WP_Rewrite::using_permalinks()` might be a more future-proof method than using the public class var. Probably doesn't really matter but it could prevent the necessity of making changes later.

Reply:
Much better. Thanks!